scripts/run_app.sh

```diff
@@ -11,7 +11,7 @@ export DM_DATA_API_URL=${DM_DATA_API_URL:=http://localhost:5000}
 export DM_DATA_API_AUTH_TOKEN=${DM_DATA_API_AUTH_TOKEN:=myToken}
 export DM_API_AUTH_TOKEN=${DM_API_AUTH_TOKEN:=myToken}
 
-export DM_SUBMISSIONS_BUCKET=${DM_SUBMISSIONS_BUCKET:=digitalmarketplace-submissions-dev-dev}
+export DM_SUBMISSIONS_BUCKET=${DM_SUBMISSIONS_BUCKET:=digitalmarketplace-documents-dev-dev}
 export DM_COMMUNICATIONS_BUCKET=${DM_COMMUNICATIONS_BUCKET:=digitalmarketplace-communications-dev-dev}
 export DM_ASSETS_URL=${DM_ASSETS_URL:=http://asset-host}
```

Comment:
I've left this comment on Rob's PR, but I think it got merged without it, so I'll copy it here:
> I think the assets URL should be the digitalmarketplace-documents-dev-dev S3 URL: https://${DM_DOCUMENTS_BUCKET}.s3-eu-west-1.amazonaws.com, since we're using a real S3 bucket for dev we can actually link directly to it from services